timelock-server/src/test/java/com/palantir/atlasdb/timelock/paxos/PaxosLeaderElectionServiceSerializationTest.java

```diff
@@ -1,63 +0,0 @@
-/*
- * (c) Copyright 2018 Palantir Technologies Inc. All rights reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package com.palantir.atlasdb.timelock.paxos;
-
-import static org.assertj.core.api.Assertions.assertThat;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.when;
-
-import java.util.Optional;
-
-import org.junit.Test;
-
-import com.fasterxml.jackson.core.JsonProcessingException;
-import com.fasterxml.jackson.databind.ObjectMapper;
-import com.fasterxml.jackson.datatype.guava.GuavaModule;
-import com.fasterxml.jackson.datatype.jdk8.Jdk8Module;
-import com.google.common.net.HostAndPort;
-import com.palantir.leader.LeaderElectionService;
-
-import io.dropwizard.jackson.Jackson;
-
-public class PaxosLeaderElectionServiceSerializationTest {
-    private static final ObjectMapper MAPPER = Jackson.newObjectMapper();
-
-    static {
-        MAPPER.registerModule(new GuavaModule());
-        MAPPER.registerModule(new Jdk8Module());
-    }
-
-    @Test
-    public void canSerializeNoSuspectedLeader() throws JsonProcessingException {
-        LeaderElectionService leaderElectionService = mock(LeaderElectionService.class);
-        when(leaderElectionService.getSuspectedLeaderInMemory()).thenReturn(Optional.empty());
```

Comment:
these tests also make no sense, it's just testing the serialisation of `HostAndPort`, and if we've bumped guava or in the case of leader block atlas, a service has bumped guava which has changed the serde, then we're screwed anyway.

Reply:
Potentially, but if we do bump guava, what then? and if we don't bump guava but a consumer does, also what then?

Reply:
In addition I suppose it's just a roundabout way of testing it, it could have just been `HostAndPortSerDeTests` 🤷‍♂

Reply:
For 1, the tests fail the version bump, we see it, and then decide either to take it and eat the break, or not to take it (more realistically not to take it immediately, for whatever reasons). This doesn't catch 2, of course - but I'd argue that 1 would be enough.

Yeah, it's indirect.